src/components/navigation/nav.tsx

```diff
@@ -0,0 +1,214 @@
+        <div className={styles.right_nav} data-testid="right-nav">
+          <nav>
+            <ul>
+              <Link to="#" className={styles.Link}>
+                <li>
+                  <FontAwesomeIcon icon="home" className={`${styles.settings}`} />
+                  &nbsp;&nbsp;Home
+                </li>
+              </Link>
+              <Link to="#" className={styles.Link}>
+                <li>
+                  <FontAwesomeIcon icon="address-book" className={`${styles.settings}`} />
+                  &nbsp;&nbsp;Contact
+                </li>
+              </Link>
+              <Link to="#" className={styles.Link}>
+                <li>
+                  <FontAwesomeIcon icon="circle-question" className={`${styles.settings}`} />
+                  &nbsp;&nbsp;About
+                </li>
+              </Link>
+              <Link to="#" className={styles.Link}>
+                <li>
+                  <FontAwesomeIcon icon="arrow-right-from-bracket" />
+                  &nbsp;&nbsp;Login
+                </li>
+              </Link>
+            </ul>
+          </nav>
+        </div>
+      )}
+      {isProfileIcon && (
+        <div className={styles.profile}>
+          <nav>
+            <ul>
+              <Link to="#" className={styles.Link}>
+                <li>
+                  <FontAwesomeIcon icon="gear" className={`${styles.settings}`} />
+                  &nbsp;&nbsp;Manage My Account
+                </li>
+              </Link>
+              <Link to="#" className={styles.Link}>
+                <li className={styles.cart}>
+                  <FontAwesomeIcon icon="cart-shopping" className={`${styles.light}`} />
+                  &nbsp;&nbsp;My Cart
+                </li>
+              </Link>
+              <Link to="#" className={styles.Link}>
+                <li>
+                  <FontAwesomeIcon icon="bag-shopping" className={`${styles.order}`} />
+                  &nbsp;&nbsp;My Orders
+                </li>
+              </Link>
+              <Link to="#" className={styles.Link}>
+                <li className={styles.upperHeart}>
+                  <FontAwesomeIcon icon="heart" className={`${styles.light}`} />
+                  &nbsp;&nbsp;Wishlist
+                </li>
+              </Link>
+              <Link to="#" className={styles.Link}>
+                <li>
+                  <FontAwesomeIcon icon="arrow-right-from-bracket" className={`${styles.logout}`} />
+                  &nbsp;&nbsp;LogOut
+                </li>
+              </Link>
+            </ul>
+          </nav>
+        </div>
+      )}
+      <div className={styles.top__bar}>
+        <i>
+          <FontAwesomeIcon
+            icon="bars"
+            className={styles.nav__toggle}
+            onClick={toggleVisibility}
+            aria-label="bars-icon"
+          />
+        </i>
+        <div className={styles.vectors}>
+          <img className={styles.logo__icon} src="/images/Vector.png" />
```

Comment:
this logo also is linked badly, remember we have concluded that we put . before /images, like this : ` <img className={styles.logo__icon} src="./images/Vector.png" />` , if not this, it will not how in production, or check it after you run build, without dot, the image can't show